src/data/faq_de.json

```diff
@@ -282,8 +282,7 @@
                                 "anchor": "rat_multiple_rat",
                                 "active": false,
                                 "textblock": [
-                                    "Nein, es kann immer nur ein Schnelltest in der App registriert werden. Ein negativer Schnelltest wird für 48 Stunden in der App angezeigt. Nach Ablauf dieser Zeit können Sie einen neuen Test registrieren. Sie haben auch die Möglichkeit, den negativen Schnelltest und das Ergebnis zu entfernen, um anschließend einen neuen Schnelltest zu registrieren. Positive Schnelltests werden nicht mehr in der App angezeigt, sobald sie geteilt wurden.",
-                                    "PCR-Tests werden unabhängig von den Schnelltests behandelt. Das heißt, Sie können jeweils einen PCR-Test und einen Schnelltest zur gleichen Zeit in der App registriert haben."
+                                    "Ja, seit Version 2.21 ist es möglich Schnelltests von Familienmitgliedern in der Corona-Warn-App zu verwalten. Pro Person gilt weiterhin: <ul><li>Ein negativer Schnelltest wird für 48 Stunden in der App angezeigt. Nach Ablauf dieser Zeit können Sie einen neuen Test registrieren. Sie haben auch die Möglichkeit, den negativen Schnelltest und das Ergebnis zu entfernen, um anschließend einen neuen Schnelltest zu registrieren.</li><li>Positive Schnelltests werden nicht mehr in der App angezeigt, sobald sie geteilt wurden.</li><li>PCR-Tests werden unabhängig von den Schnelltests behandelt. Das heißt, Sie können jeweils einen PCR-Test und einen Schnelltest zur gleichen Zeit in der App registriert haben.</li></ul>"

```

Comment:
The last sentence is still no really clear, whether this applies to all tests in general or only tests from one person.

Reply:
In my opinion this is already stated clear enough, as the list is introduced with the information that the content of the list applies per person and not in total. Before, when the sentence was in a seperate paragraph, I could understand the confusion, which is why I ended up listing them together. 

Alternatively I could change the last sentence to "Das heißt, Sie können **pro Person** jeweils einen PCR-Test und einen Schnelltest zur gleichen Zeit in der App registriert haben", but this would lead to a duplication regarding the list introduction.

Reply:
@larswmh I agree, should be clear enough with the first sentence.
Feel free to mark this comment and the corresponding comment in English as resolved.

Reply:
You could put "Pro Person" in bold text to emphasize.

Reply:
> You could put "Pro Person" in bold text to emphasize.

Will do that, thanks for the suggestion!